5/docker-entrypoint.sh

```diff
@@ -11,8 +11,6 @@ if [ "$1" = 'kibana' ]; then
 	if [ "$ELASTICSEARCH_URL" ]; then
 		sed -ri "s!^(\#\s*)?(elasticsearch\.url:).*!\2 '$ELASTICSEARCH_URL'!" /etc/kibana/kibana.yml
 	fi
-
-	set -- gosu kibana tini -- "$@"
```

Comment:
Look like we are losing `tini` too. We should keep that; signals are important (https://github.com/docker-library/kibana/issues/21).